static/src/javascripts-legacy/projects/common/modules/email/email-article.js

```diff
@@ -205,16 +210,41 @@ define([
 
                 storage.session.set('email-sign-up-seen', 'true');
             }
-        };
+        }
 
     return {
         init: function () {
             if (emailRunChecks.allEmailCanRun()) {
                 // First we need to check the user's email subscriptions
                 // so we don't insert the sign-up if they've already subscribed
                 emailRunChecks.getUserEmailSubscriptions().then(function () {
-                    // Get the first list that is allowed on this page
-                    addListToPage(find(listConfigs, emailRunChecks.listCanRun));
+                    if (ab.isParticipating({id: 'TailorRecommendedEmail'}) &&
+                        ab.isInVariant('TailorRecommendedEmail', 'tailor-recommended')) {
+                        var bwidCookie = cookies.get('bwid') || false;
+
+                        if (bwidCookie) {
+                            var cachedTailorResponse = cookies.get('GU_TAILOR_EMAIL') || false;
```

Comment:
You can add a check for local storage support in the a/b test canRun function.  See https://github.com/guardian/frontend/blob/f19a107f83dc4e8467be15ac1fcddb69e83b6de8/static/src/javascripts/projects/common/modules/experiments/tests/new-user-adverts-disabled.js#L23